internal/config/options.go

```diff
@@ -188,148 +187,68 @@ var defaultOptions = Options{
 	GRPCAddr:                ":443",
 	GRPCClientTimeout:       10 * time.Second, // Try to withstand transient service failures for a single request
 	GRPCClientDNSRoundRobin: true,
+	viper:                   viper.New(),
```

Comment:
Since viper.New() returns pointer, don't we wind up sharing a single viper instance everywhere if we start with a copy of this struct?  I avoided this pattern because of that assumption...am I forgetting how that works?

Reply:
Yes, you totally had the right to begin with. I tried to remove the NewXXX() but ended up going back to it. My bad.

Reply:
I spent a few minutes trying to remember I didn't use exactly the same pattern  😆